### PR TITLE
[common] Reduce code duplication

### PR DESCRIPTION
**Pull request by mvieth**
Please see commit messages. Was originally part of pull request #4452

### CODE REVIEW

File: test/common/test_common.cpp [hunk 1]
```diff
@@ -531,6 +531,55 @@ TEST (PCL, HasField)
...
 
+TEST (PCL, GetMinMax3D)
+{
+  pcl::PointCloud<pcl::PointXYZ> cloud;
+  cloud.emplace_back ( 0.0f,      0.0f,  0.0f);
+  cloud.emplace_back (10.0f, -10000.0f,  1.0f);
+  cloud.emplace_back ( 5.0f,      5.0f,  0.0f);
+  cloud.emplace_back (-5.0f,      0.0f, -0.5f);
+
+  pcl::PointIndices pindices;
```
Comment (kunaltyagi) [inline test/common/test_common.cpp:542]: Shall we move the indices variables closer to the tests?

Comment (mvieth) [reply test/common/test_common.cpp:542]: My idea was to first create all the needed objects, and then do the tests. But I can move it around if you like that better.

**Review by larshg — APPROVED** [review]

File: test/common/test_common.cpp (hunk 1 above)
Comment (kunaltyagi) [reply test/common/test_common.cpp:542]: Please do. Since not all the tests use the same variables, it'll be easier to read

**Review by larshg — APPROVED** [review]

File: test/common/test_common.cpp (hunk 1 above)
Comment (mvieth) [reply test/common/test_common.cpp]: Done

**Review by kunaltyagi — APPROVED** [review]

**Pull request merged**